new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleNginxValidation.java

```diff
@@ -628,4 +673,149 @@ private void checkServiceCreated(String serviceName) {
 
   }
 
+  /**
+   * Check pod does not exist.
+   *
+   * @param podName pod name to check
+   */
+  private void checkPodDoesNotExist(String podName) {
+    withStandardRetryPolicy
+        .conditionEvaluationListener(
+            condition -> logger.info("Waiting for pod {0} to be removed in namespace {1} "
+                    + "(elapsed time {2}ms, remaining time {3}ms)",
+                podName,
+                domainNamespace,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(assertDoesNotThrow(() -> podDoesNotExist(podName, null, domainNamespace),
+            String.format("Pod %s still exists in namespace %s", podName, domainNamespace)));
+  }
+
+  /** Scale the WebLogic cluster of the domain to numberOfServers in the specified domain namespace.
+   *  Verify the pods are created or deleted depending on the numberOfServers. Also verify
+   *  NGINX can access the sample apps to all the servers after the scale operation.
+   *
+   * @param domainUid the domain with domainUid which will be scaled
+   * @param domainNamespace the domain namespace the domain resides
+   * @param clusterName the WebLogic cluster name in the domain to be scaled
+   * @param replicasBeforeScale the replicas of the WebLogic cluster before the scale
+   * @param numberOfServers the number of servers to be scaled to
+   * @return the replicas of the WebLogic cluster after the scale
+   */
+  private int scaleAndVerifyDomain(String domainUid,
+                                   String domainNamespace,
+                                   String clusterName,
+                                   int replicasBeforeScale,
+                                   int numberOfServers) {
+
+    String manageServerPodNamePrefix = domainUid + "-" + clusterName + "-managed-server";
+
+    // scale the cluster in the domain
+    assertThat(assertDoesNotThrow(() -> scaleDomain(domainUid, domainNamespace, clusterName, numberOfServers)))
+        .as(String.format("Verify scale the cluster %s of domain %s in namespace %s",
```

Comment:
this message will be printed on a successful scale? If yes then it needs to say "cluster %s of domain %s in namespace %s scaled to %s(replicasBeforeScale+numberOfServers)"

Reply:
no, this message is describing what the assertion does, not a message for a successful run.